homework4/task3/tests/test_get_print_output.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from project3__get_print_output.task3_get_print_output import my_precious_logger
+
+
+@pytest.mark.parametrize(
+    ("string", "expected_result"),
+    [
+        ("OK", "OK"),
+        ("", ""),
+        ("ERROR", "ERROR"),
+    ],
+)
+def test_my_precious_logger_positive(capsys, string, expected_result):
+    my_precious_logger(string)
+    result = capsys.readouterr()
+
+    assert result.out == expected_result
```

Comment:
Which side effects you function does?  Do we test them all?

Reply:
You mean to assert both result.out and result.err? Fixed